opencti-platform/opencti-front/src/private/components/data/stream/StreamCollectionEdition.jsx

```diff
@@ -76,56 +71,43 @@ const streamCollectionValidation = (t) => Yup.object().shape({
   name: Yup.string().required(t('This field is required')),
   description: Yup.string().nullable(),
   stream_public: Yup.bool().nullable(),
+  authorized_members: Yup.array().nullable(),
 });
 
-const groupMutationRelationAdd = graphql`
-  mutation StreamCollectionEditionGroupAddMutation($id: ID!, $groupId: ID!) {
-    streamCollectionEdit(id: $id) {
-      addGroup(id: $groupId) {
-        ...StreamCollectionEdition_streamCollection
-      }
-    }
-  }
-`;
-
-const groupMutationRelationDelete = graphql`
-  mutation StreamCollectionEditionGroupDeleteMutation($id: ID!, $groupId: ID!) {
-    streamCollectionEdit(id: $id) {
-      deleteGroup(id: $groupId) {
-        ...StreamCollectionEdition_streamCollection
-      }
-    }
-  }
-`;
-
-const StreamCollectionEditionContainer = (props) => {
-  const { t, classes, handleClose, streamCollection } = props;
-  const groups = pipe(
-    pathOr([], ['groups']),
-    map((n) => ({
-      label: n.name,
-      value: n.id,
-    })),
-  )(streamCollection);
-  const initialValues = { ...streamCollection };
-  initialValues.groups = groups;
+const StreamCollectionEditionContainer = ({ handleClose, streamCollection }) => {
+  const classes = useStyles();
+  const { t } = useFormatter();
+  const authorizedMembers = streamCollection.authorized_members?.map(({ id, name }) => ({ value: id, label: name })) ?? [];
```

Comment:
Could we have a utils in edition.js like for the rest of the fields ?

Reply:
Done